docs/user/authoring-input-modes.rst

```diff
@@ -0,0 +1,69 @@
+Authoring Input Modes
+=====================
+
+# Overview
+
+A number of components provide multiple authoring modes.
+
+As a concrete example, the waves generated by the ShapeFFT component can be added to the world in multiple ways, including:
+
+* **Global** - waves simply added everywhere in the world
+* **Painted** - waves are painted down by hand in particular areas
+* **Spline** - waves are added along a user defined spline
+
+These are selected using the **Input Mode** dropdown on the ShapeFFT component, which display the valid modes for each component.
+
+The following sections describe each mode.
+
+
+# Global Mode
```

Comment:
See other files for correct heading syntax. And remember there is a live build you can view to see formatting: https://crest.readthedocs.io/en/feature-painted-waves/user/authoring-input-modes.html

Reply:
Thanks will finalize docs with this and other notes